### ISSUE TITLE
[bug report] El toolip: when the click event triggers confirm, there is a focus problem after confirm completes processing, and toolip will appear on its own.

### ISSUE THREAD

**Issue by chenxch** [ElemeFE/element#19301]
### Element UI version
2.13.1

### OS/Browsers version
win10/chrome79.0.3945.79

### Vue version
2.6.11

### Reproduction Link
https://codepen.io/chenxch/pen/ExVvjob

### Steps to reproduce
1、点击tooltip包裹的按钮
2、处理弹出框，取消或者确定
3、按钮的tooltip就自动出现了

### What is Expected?
confirm处理后，tooltip不要自动focus当初提示

### What is actually happening?
confirm处理后，tooltip自动focus了

**Comment by migueling** [comment]
Because the MessageBox returns focus to the last active element when closed, a workaround for me was to call document.activeElement.blur() before this.$confirm() to change focus from button to body.

**Comment by chenxch** [comment]
@migueling thk.

**Comment by 5265liu** [comment]
thanks for your answer,  but i think this is a bug

**Comment by migueling** [comment]
> thanks for your answer, but i think this is a bug

Yes, I also think it's a bug, what I commented was just a workaround that I used.

**Comment by chenxch** [comment]
I hope the element UI team can fix this problem.

**Comment by Huawanjiu** [comment]
> Because the MessageBox returns focus to the last active element when closed, a workaround for me was to call document.activeElement.blur() before this.$confirm() to change focus from button to body.

thanks, this works for me, call blur method on el-tooltip element before this.$confirm().
